pymatgen_diffusion/neb/full_path_mapper.py

```diff
@@ -234,6 +239,8 @@ def _get_pos_and_migration_path(self, u, v, w):
             lattice=self.only_sites.lattice,
         )
         # Positions might be useful for plotting
+        edge["isite"] = i_site
+        edge["esite"] = e_site
```

Comment:
I think if you use `MigrationHop.__eq__` to compare the edges you no longer need these.  They are also stored one level down as attributes of `d['hop']`.

Reply:
These are added so that when a unique_hops dictionary is stored in a database the isite  and esite can be easily reconstructed. That does not happen with how the hop object is serialized currently. I was trying to add a way to create the hop object from a unique_hop dictionary that has been stored in a database.